net/mptcp/Kconfig

```diff
@@ -2,11 +2,11 @@
 # MPTCP configuration
 #
 config MPTCP
-        bool "MPTCP protocol"
-        depends on (IPV6=y || IPV6=n)
-        ---help---
-          This replaces the normal TCP stack with a Multipath TCP stack,
-          able to use several paths at once.
+	bool "MPTCP protocol"
+	depends on (IPV6=y || IPV6=n)
+	---help---
+	  This replaces the normal TCP stack with a Multipath TCP stack,
+	  able to use several paths at once.
```

Comment:
Please revert these changes.

Reply:
Ok, I will revert this changes. I just tried to stick as close as possible to the coding style (section 10) mentioned here: https://www.kernel.org/doc/html/v4.19/process/coding-style.html
>  Lines under a config definition are indented with one tab, while help text is indented an additional two spaces